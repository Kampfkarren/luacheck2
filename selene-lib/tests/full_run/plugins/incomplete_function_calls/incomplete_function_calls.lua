local function my_function(_x, _y, _z) end

my_function(1, 2, 3)
my_function(1)
bad_function()
