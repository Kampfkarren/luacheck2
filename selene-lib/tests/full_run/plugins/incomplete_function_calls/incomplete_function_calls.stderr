warning[plugin_incomplete_function_calls]: not enough arguments. expected 3, got 1
  ┌─ incomplete_function_calls.lua:4:1
  │
4 │ my_function(1)
  │ ^^^^^^^^^^^^^^

error[undefined_variable]: `bad_function` is not defined
  ┌─ incomplete_function_calls.lua:5:1
  │
5 │ bad_function()
  │ ^^^^^^^^^^^^

